scripts/bump_version.py

```
@@ -0,0 +1,388 @@
+RS_SOURCE_ID = Path("modelexpress_server/src/p2p/source_identity.rs")
+
+# Directories pruned from the mx_version discovery walk.
+PRUNE_DIRS = {".git", "target", ".venv", "venv", "__pycache__", "node_modules", ".mypy_cache", ".pytest_cache"}
+# File extensions scanned for mx_version literals.
+SCAN_SUFFIXES = {".rs", ".py", ".md", ".yaml", ".yml", ".json"}
+
+SEMVER_RE = re.compile(r"^\d+\.\d+\.\d+([.\-+].+)?$")
+HEX16 = r"[0-9a-f]{16}"
+
+
+def fail(message: str) -> "NoReturn":  # type: ignore[name-defined]
```

Ruff (0.15.15)

[error] Undefined name `NoReturn`

(F821)